# ros 时间
Header header

#imu可用性
bool imu_vaild

#lidar可用性
bool lidar_vaild

#rtk可用性
bool rtk_vaild

#leg可用性
bool leg_vaild
